### ISSUE TITLE
comma-spacing error is thrown inside of template literal

### ISSUE THREAD

**Issue by KevinGrandon** [eslint/eslint#2533]
Some javascript like:

```js
var attr = 'hello';
var aString = `somestring=${attr},${attr}`;
console.log(aString);
```

Produces the following error:

```
error  A space is required after ','  comma-spacing
```

I believe this is valid syntax inside of a template literal, and there should be no errors thrown.

**Comment by gyandeeps** [comment]
Can you paste the output for the above code and also what kind of parser you are using?
Also did you try it with version 0.21.0?

**Comment by KevinGrandon** [comment]
We are currently running 0.21.0, and using gulp-eslint.

**Comment by gyandeeps** [comment]
In your project do you setup eslint configurations or inside your gulp file when you use the gulp-eslint plugin?
if yes, can you share that.

**Comment by gyandeeps** [comment]
If you are using `babel-eslint` as the parser then its an issue with that project.
reference: https://github.com/eslint/eslint/issues/2497

**Comment by KevinGrandon** [comment]
Great, thanks for the info!

For reference, looks like this is tracked in: https://github.com/babel/babel-eslint/issues/31

**Comment by hzoo** [comment]
Working on a fix for this.
